docs/api-docs/getting-started/rest-api-authentication.mdx

```diff
@@ -215,6 +215,9 @@ All OAuth scopes except `default` provide `read-only` permissions scopes so that
 | Information & Settings | read-only | `store_v2_information_read_only` | View general store information and settings | [Store Information V2](/docs/rest-management/store-information#get-store-information) <br />[Time V2](/docs/rest-management/store-information#get-system-timestamp) <br />[Settings V3](/docs/rest-management/settings)|
 | Marketing | modify | `store_v2_marketing` | View and modify marketing information | [Coupons V2](/docs/rest-content/marketing/coupons#get-all-coupons) <br />[Gift Certificates V2](/docs/rest-content/marketing/gift-certificates#get-all-gift-certificates) <br />[Banners V2](/docs/rest-content/marketing/banners#get-all-banners) <br />[Promotions V3](/beta/promotions/rest-management/coupon-codes-bulk)|
 | Marketing | read-only | `store_v2_marketing_read_only` | View marketing information | [Coupons V2](/docs/rest-content/marketing/coupons#get-all-coupons) <br />[Gift Certificates V2](/docs/rest-content/marketing/gift-certificates#get-all-gift-certificates) <br />[Banners V2](/docs/rest-content/marketing/banners#get-all-banners) |
+| Metafield Ownership | manage | `` | Manage which API accounts own metafields. |  |
```

Comment:
@kskomro could you please:
* provide the parameter names
* review the descriptions
* let me know which APIs metafield ownership scopes apply to

Reply:
@slsriehl 

- what do you mean by parameter names? If policies names then: store_metafield_change_owner (Metafield Ownership) and store_metafield_write_all (Metafields Access)
- for descriptions:
Metafield Ownership - manage from what I see means also viewing in platform, but to view ownership one does not need this permission, so I would say: `Change API client owner of metafieldsz
Metafields Access | standard - it is fine
Metafields Access | full - I would remove `Create` part since one does not need this scope to create metafield. Every api client can and users with scope StorePermission.MANAGE_SETTINGS (don't know what's the name in UI)
- Ownership applies really only to graphql since we don't have endpoint in v3 to change ownership. Metafields Access full - applies both in Admin GraphQL and Admin v3.